tezos_interop/tezos_interop.rei

```diff
@@ -151,6 +153,8 @@ module Consensus: {
   };
   let listen_operations:
     (~context: Context.t, ~on_operation: operation => unit) => unit;
+  let fetch_validators:
+    (~context: Context.t) => Lwt.t(result(list(Secret.t), string));
```

Comment:
```suggestion
    (~context: Context.t) => Lwt.t(result(list(Key.t), string));
```

There is no way the storage can return `Secret.t`, as they're never stored on Tezos to begin with, probably this should be Key.t